build/args/all.gn

```diff
@@ -33,3 +33,7 @@ is_cfi = false
 allow_runtime_configurable_key_storage = true
 
 enable_cet_shadow_stack = false
+
+# TODO(deepak1556): remove once https://bugs.chromium.org/p/chromium/issues/detail?id=1241610
+# is fixed.
+enable_blink_heap_use_v8_oilpan = false
```

Comment:
terrifying, is there some way we can ensure we don't release with this disabled?

Reply:
Looks like https://chromium.googlesource.com/v8/v8/+/4853c3ad81b178811814e7ab8fc28ecafe911e57 addresses the issue and should hit the chromium roll tomorrow.  I'll check then and see if I can re-enable in #30676

Reply:
> is there some way we can ensure we don't release with this disabled?

Can you expand on why it is bad to disable this flag ? I see that this feature was only enabled recently https://chromium-review.googlesource.com/c/chromium/src/+/3089152, so shouldn't the old code path be still fine to use ?

But it is great that upstream fixed the regression as well, we can turn it on again in the next roll.